tests/framework/helpers/StringHelperTest.php

```diff
@@ -19,7 +19,9 @@ class StringHelperTest extends TestCase
     protected function setUp()
     {
         parent::setUp();
-        $this->mockApplication();
+
+        // destroy application, Helper must work without Yii::$app
+        $this->destroyApplication();
```

Comment:
shouldnt we extend from a PhpUnit base class in which no application is setup?

Reply:
@dynasource `yiiunit\TestCase` dose not mock application by default. `destroyApplication()` is used only to make sure that `Yii::$app` is not setup as leftovers after previous tests.